libnymea-zigbee/zcl/general/zigbeeclusterlevelcontrol.cpp

```diff
@@ -56,12 +57,12 @@ ZigbeeClusterReply *ZigbeeClusterLevelControl::commandMove(ZigbeeClusterLevelCon
     return executeClusterCommand(ZigbeeClusterLevelControl::CommandMove, payload);
 }
 
-ZigbeeClusterReply *ZigbeeClusterLevelControl::commandStep(ZigbeeClusterLevelControl::FadeMode fadeMode, quint8 stepSize, quint16 transitionTime)
+ZigbeeClusterReply *ZigbeeClusterLevelControl::commandStep(StepMode stepMode, quint8 stepSize, quint16 transitionTime)
```

Comment:
FYI, there seems to be a bug in the Spec. The parameter description says "StepMode", but the description of the actual content of this register says "FadeMode". Given the command is named "Step", "FadeMode" really seems to be a bug in the spec and "StepMode" is the correct one.

Reply:
Interesting...cannot remember, but makes sense to me